src/Sulu/Component/Content/Repository/ContentRepository.php

```diff
@@ -527,7 +527,7 @@ private function getQueryBuilder($locale = null, $locales = [], ?UserInterface $
         return $queryBuilder;
     }
 
-    private function getPropertyName($propertyName, $locale)
+    private function getPropertyName(string $propertyName, $locale): string
```

Comment:
Not a BC break as it would crash when the method does not return a string (which is what the failing tests complain about. 
Parameter is always a string (see usage above).